### ISSUE TITLE
required parameter $size is missing in call to mixin make-col

### ISSUE THREAD

**Issue by MarzChew** [twbs/bootstrap#19475]
Hi I get an compilation error "required parameter $size is missing in call to mixin make-col" since I updated my bootstrap v4 version.

I don't succeed to identify where it comes from.
Any clue ?
Thanks

**Comment by cvrebert** [comment]
Unable to reproduce with current `v4-dev` branch.
For troubleshooting help, please use StackOverflow or similar; see https://github.com/twbs/bootstrap/blob/master/README.md#community for links.

**Comment by exadeci** [comment]
Had the same issue, `make-col` used to behave like `make-col-ready` does now but it now takes a `$size` argument.

So now `make-col-ready` has to be used to replace `make-col` and `make-col` has to be used to replace `make-col-span`

It was only documented from this commit https://github.com/twbs/bootstrap/commit/a8879c8f82ec3debb8e225844dc4561e2900beda

**Comment by p-h-i-l** [comment]
Having the same issue after upgrading to Alpha 3. 
I replaced all occurrences of `make-col`with `make-col-ready` and all occurrences of `make-col-span`with `make-col(X)`. But the size argument for make-col-ready is still missing... 👎 

Any idea?

**Comment by Zyphrax** [comment]
The mixin `make-col-ready` requires a `size` argument, but doesn't use it and it isn't mentioned in the documentation.

The docs say do this:
```
  @include make-col-ready();
  @include make-col(8);
```

At the moment you have to do this:
```
  @include make-col-ready(8);
  @include make-col(8);
```

What is the benefit of separating `make-col-ready` and `make-col`?
Why not just one mixin `make-col`?

Perhaps any necessary distinction could be made using an extra argument.

**Comment by ziemkowski** [comment]
@cvrebert That's because this was fixed after Alpha 3 via @4f947f4. 

It seems this issue should be added to the `v4.0.0-alpha.4` milestone (the pull request #20413 is in the a4 ship list, but there isn't an issue attached).
